### DISCUSSION TITLE
SignUp doesn't work anymore

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#6336] [Questions] [answered]
Hi 👋🏻 

I tried to have a ``public.profile`` table sharing data with the ``auth.users`` one, using steps in the documentation (https://supabase.com/docs/guides/auth/managing-user-data#advanced-techniques), but it looks like my users are not registered anymore. 

![image](https://user-images.githubusercontent.com/25107942/162090766-9b6f89dd-8bbd-41f4-addd-652a218601e7.png)

My code:

```ts
import { createClient } from '@supabase/supabase-js';
import 'dotenv/config';

const supabase = createClient(
    process.env.VITE_SUPABASE_URL as string,
    process.env.VITE_SUPABASE_KEY as string
);

supabase.auth.signUp({
    email: 'admin@admin.com',
    password: 'admin123',
});

console.log('Migrations done:', supabase.auth.user());
```

Any idea?

**Comment by GaryAustin1** [comment, marked as answer]
This Usually would be caused by your trigger function getting an error.  That will stop user being created.  Any errors returned like “500”?  Also there might be info in the database log. 
I notice you also console.log a message about Migrations.  Is it possible one of the tables has an issue after migration?

**Reply by Olyno** [reply]
Ah indeed I had a mistake, the type of the id was not the right one 😅 

Concerning the migration, it's just a file that I created and where I put the code above, it's nothing more. It is not related to ``supabase-cli``.
